packages/guides/src/Nodes/Inline/ImageInlineNode.php

```diff
@@ -20,8 +20,11 @@ final class ImageInlineNode extends InlineNode
 {
     public const TYPE = 'image';
 
-    public function __construct(private readonly string $url, private readonly string $altText)
-    {
+    public function __construct(
+        private readonly string $url,
+        private readonly string $altText,
+        private readonly string $title,
```

Comment:
Adding a new constructor parameter is a backwards compatible break. Please make this an optional value which is by default `null`

Reply:
Done